addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -618,5 +619,16 @@
     </table>
 </template>
 
+<template id="forum_sign_up_cta" name="Sign-Up Call to Action">
```

Comment:
Same remark as earlier, would call it ""sign_up_cta" as we already have the module prefix.